cli/src/lib.rs

```diff
@@ -3707,8 +3703,7 @@ fn create_idl_account(
 ) -> Result<Pubkey> {
     // Misc.
     let idl_address = IdlAccount::address(program_id);
-    let keypair = solana_sdk::signature::read_keypair_file(keypair_path)
-        .map_err(|_| anyhow!("Unable to read keypair file"))?;
+    let keypair = get_keypair(&cfg.provider.wallet.to_string())?;
```

Comment:
This keeps getting resolved, but it's not. It should either use the `keypair_path` parameter (rather than getting the configured wallet path from `cfg.provider.wallet`), or the `keypair_path` parameter should be removed.

Reply:
Got it, sorry I didn't realize `keypair_path` was dead code until now.